bakery/mgostorage/rootkey.go

```diff
@@ -208,14 +208,14 @@ func (s *RootKeys) setCurrent(policy Policy, key rootKey) {
 	s.current[policy] = key
 }
 
-type rootKeyStorage struct {
+type storage struct {
```

Comment:
Note that it's bakery.Storage, not just "storage". It is the only storage that the bakery code writes to, and the old storage type was also just named "storage".